CLI.md

```diff
@@ -389,6 +403,7 @@ Add or modify a validator (admin only)
 * `--votes <VOTES>` — Voting power
 
   Default value: `1`
+* `--force` — Set the validator even if the version or genesis config check fails
```

Comment:
`--skip-online-test` ?

Reply:
Currently it does the check even with `--force`, but prints the warning.
But skipping the whole check would also simplify the code. I'll change it. :innocent: